playbooks/roles/notify-failure-rocket/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Clone ardana-bm-info repository
+  git:
+    repo: "{{ rc_auth_info_git_url }}"
+    dest: "{{ rc_auth_info_dir }}"
+    version: "master"
+    force: yes
+
+- name: Load rocketchat auth variables
+  include_vars: "{{ rc_auth_info_dir }}/rc.yml"
+
+- name: Notify RocketChat of the failure to clean up
+  rocketchat:
+    domain: "{{ rc_domain }}"
+    token: "{{ rc_token }}"
+    username: "Jenkins"
+    channel: "{{ rc_channel }}"
+    validate_certs: no
+    protocol: "{{ rc_protocol }}"
+    attachments:
```

Comment:
Why not simply using "msg?"
```msg: "Failure in Job teardown for {{ lookup('env','BRANCH_NAME') }}. See log: {{ lookup('env','RUN_DISPLAY_URL') }}" ```

Reply:
Looks nicer!
There is a lot of information there as to have everything in one line. Using attachment not only shows prominently in the chat but has the link embedded and the information is readable.

Reply:
Fine for me then :)